tflib/imagetest/helm/main.tf

```diff
@@ -0,0 +1,60 @@
+variable "name" {
```

Comment:
nit, not blocking: might be worth splitting into variables.tf if we have this many